docs/troubleshooting.rst

```diff
@@ -13,9 +13,12 @@ Troubleshooting
 Windows Issues
 --------------
 
-* Some people have reported issues using git bash; try using the Command Terminal instead.
+* Some people have reported issues using git bash; try using the
```

Comment:
here we could also recommend Windows Subsystem for Linux, which I've found quite useful!

Reply:
I've never used it but I've heard good things! Do you have an idea for sentence or two that would cover it?